Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/PublicBlogController.java

```diff
@@ -0,0 +1,61 @@
+package edu.asu.spring.quadriga.web.publicwebsite;
+
+import java.security.Principal;
+import java.util.*;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.spring.quadriga.domain.workbench.IProject;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.workbench.IRetrieveProjectManager;
+
+@PropertySource(value = "classpath:/user.properties")
+@Controller
+public class PublicBlogController {
+	@Autowired 
+	private IRetrieveProjectManager projectManager;
+	
+	private IProject getProjectDetails(String name) throws QuadrigaStorageException{
+		return projectManager.getProjectDetailsByUnixName(name);
```

Comment:
why do you have a separate method for this? It's just one line, I'd put that directly in line 40.